From 0bea83623de441e020dfaec3250f2784d1c73c7a Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 19 Nov 2014 16:12:56 -0500
Subject: [ethernet] be2net: fix a sparse warning in be_cmd_modify_eqd()

Message-id: <1416413587-12091-9-git-send-email-ivecera@redhat.com>
Patchwork-id: 100331
O-Subject: [RHEL7.1 PATCH 08/19] be2net: fix a sparse warning in be_cmd_modify_eqd()
Bugzilla: 1165755
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: #1165755

Upstream commit(s):
commit b502ae8d9cbbbf4ad822571537b9e17173d9c8b8
Author: Kalesh AP <kalesh.purayil@emulex.com>
Date:   Fri Sep 19 15:46:51 2014 +0530

    be2net: fix a sparse warning in be_cmd_modify_eqd()

    This patch fixes a sparse warning about missing static declaration that was
    introduced by the following commit:

    fixes: 936767039cdf ("be2net: send a max of 8 EQs to be_cmd_modify_eqd() on Lancer")

    Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_cmds.c b/drivers/net/ethernet/emulex/benet/be_cmds.c
index cd213d9..2874c0b 100644
--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -1783,8 +1783,8 @@ err:
 /* set the EQ delay interval of an EQ to specified value
  * Uses async mcc
  */
-int __be_cmd_modify_eqd(struct be_adapter *adapter, struct be_set_eqd *set_eqd,
-   int num)
+static int __be_cmd_modify_eqd(struct be_adapter *adapter,
+          struct be_set_eqd *set_eqd, int num)
 {
  struct be_mcc_wrb *wrb;
  struct be_cmd_req_modify_eq_delay *req;
-- 
1.7.1